pa_matchbatch.sh

```diff
@@ -222,6 +226,27 @@ cloneOrFetch() {
 	fi
 }
 
+readRuntimeVersion() {
```

Comment:
Would it make sense to fold this into getFixVerFor()? Or is the intent different?

Reply:
The intent here is a bit different, the version is read from puppet-agent (puppet-runtime.json), the idea being to check all the changes between the version included in the last release (tag) and the current version.